graspologic/plot/plot.py

```diff
@@ -673,6 +677,253 @@ def pairplot(
     return pairs
 
 
+def plot_ellipse(
+    X,
+    Y_,
+    means,
+    covariances,
+    ax,
+    j,
+    k,
+    colors,
```

Comment:
@loftusa  i just think that since it is not an outward facing function, and for ease of readability as an internal function it would be easier this way?